src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -347,18 +352,19 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
         * @param c is a Contact record from trigger new.
         * @param oldContact is a Contact from trigger old.
         * @param triggerAction which trigger event (BeforeInsert, AfterInsert, etc.).
-        * @param numOfContactsLeft is an interger that represents the number of non deceased Contacts associated
-        * with the Account.
-        */    
-        private void handleUpdateData(Contact c, Contact oldContact, TDTM_Runnable.Action triggerAction, Integer numOfContactsLeft) {    
+        * @param numOfContactsLeft is an interger that represents the number of non deceased Contacts or Contacts
+        * that are not marked "Excluded_from_Household_Naming__c" associated with the Account.
+        */
+        private void handleUpdateData(Contact c, Contact oldContact, TDTM_Runnable.Action triggerAction, Integer numOfContactsLeft) {
             Boolean conAccountIdNotNull = (c.AccountId != NULL);
             Boolean oldConAccountIdNotNull = (oldContact.AccountId != NULL);
             Boolean conOwnerChanged = (c.OwnerId != oldContact.OwnerId);
             Boolean conLastNameChanged = (c.LastName != oldContact.LastName);
             Boolean conFirstNameChanged = (c.FirstName != oldContact.FirstName);
             Boolean conExcludeFromHHNameChanged = (c.Exclude_from_Household_Name__c != oldContact.Exclude_from_Household_Name__c);
+            Boolean conDeceasedHasChanged = (c.Deceased__c != oldContact.Deceased__c);
 
-            //If Account has changed :  (1) Update the Account's name (2) Move Opportunity
+            //If Account has changed: (1) Update the Account's name (2) Move Opportunity
             if (c.AccountId != oldContact.AccountId) {
                 if(conAccountIdNotNull) {
                     accountIdToUpdate.add(c.AccountId);
```

Comment:
NIT: like the one below, can we pluralize this? (line 370, not sure why github won't let me comment on the incoming side...and on line 343